types_webgl_wasm.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build js,wasm
+
+package gl
+
+import "syscall/js"
+
+type Enum int
+
+type Attrib struct {
+	Value int
+}
+
+type Program struct {
+	js.Value
+}
+
+type Shader struct {
+	js.Value
+}
+
+type Buffer struct {
+	js.Value
+}
+
+type Framebuffer struct {
+	js.Value
+}
+
+type Renderbuffer struct {
+	js.Value
+}
+
+type Texture struct {
+	js.Value
+}
+
+type Uniform struct {
+	js.Value
+}
+
+var NoAttrib = Attrib{0}
```

Comment:
Here could use zero value vars as well I think `var NoAttrib Attrib`

Reply:
Indeed, just corrected that.